README.md

```diff
@@ -49,6 +49,26 @@ Run the following commands, very similarly to how [.travis.yml](.travis.yml) doe
 1. `./gradlew createDB -PdbName=mifostenant-default`
 1. `./gradlew clean integrationTest`
 
+
+Instructions to run using Docker and docker-compose
+===================================================
+
+It is possible to do a 'one-touch' installation of Fineract using docker-compose
+
+  Prerequisites:
+  * docker and docker-compose installed on your machine
+
+
+  Installing a new Fineract instance:
+
+  * Clone the Fineract Github repository
+  * Navigate to the docker directory
```

Comment:
This is no longer needed now, right? Remove?

Reply:
Yes, will remove the instruction to navigate to the docker directory